Anticheat/src/init.lua

```diff
@@ -60,6 +61,8 @@ Anticheat.ChecksEnabled = {
 	InvalidDrop = true, -- Dropping tools that don't have CanBeDropped
 	ToolDeletion = true, -- Stop the client from deleting tools (Incompatible with any usage of tool.Parent = nil, use :Destroy() instead)
 	FEGodMode = true, -- God mod achieved by deleting their Humanoid on the server and creating a fake one on the client
+	PreventParentAccroutrements = true, -- If players can drop and/or delete hats(and other accessories). If you have hat drop scripts turn this to false. Up to 2017 you could drop hats via the = key however this was removed. If you have a custom hat drop script set this to false.
```

Comment:
says if you have hat scripts turn this off 2 times

Reply:
Ok fixed it.